docs/faqs.md

```diff
@@ -409,3 +409,9 @@ merging to the master branch. Feel free to contribute to a new Github Action tha
 
 No. The [pylint](v2_linter.md) has an important role of keeping any recipe prepared for [Conan v2 migration](v2_migration.md). In case you are having
 difficult to understand [linter errors](linters.md), please comment on your pull request about the problem to receive help from the community.
+
+## How long can I be inactive before being removed from the authorized users list?
```

Comment:
Might be better to right this next to request access? 

https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md#request-access

Reply:
agree 👍

Reply:
I think this is way too short. we will lose some contributors with this change. Maybe something like 12 months could be better? (all in all, I don't understand what cost you are trying to mitigate)

Reply:
Moved, please, read again.